### PR TITLE
Not allow add function field that has not setter

### PR DESCRIPTION
**Pull request by raimonesteve**
#163209

### CODE REVIEW

File: mass_editing.py [hunk 1]
```diff
@@ -99,6 +99,21 @@ class MassEditFields(ModelSQL):
...
+    def validate(cls, fields):
+        super().validate(fields)
+        for _field in fields:
+            _field.check_field()
+
+    def check_field(self):
+        Model = Pool().get(self.field.model.model)
+
+        _field = Model._fields.get(self.field.name)
+        print(self.field.name, _field)
```
Comment (albertca) [inline mass_editing.py]: This print should be removed.

File: message.xml [hunk 2]
```diff
@@ -6,6 +6,9 @@ this repository contains the full copyright notices and license terms. -->
         <record model="ir.message" id="not_modelsql">
             <field name="text">Model "%(model)s" does not store information to an SQL table.</field>
         </record>
+        <record model="ir.message" id="msg_error_setter">
+            <field name="text">Can not add the field "%(name)s" because is a function field without setter.</field>
```
Comment (albertca) [inline message.xml]: because is -> because it is

**Review by albertca — APPROVED** [review]
Apart from the minor comments, it is ok for me

File: mass_editing.py (hunk 1 above)
Comment (raimonesteve) [reply mass_editing.py]: 1ba7bdee1f26e4dd57197db15920ff3fb79033ee

File: message.xml (hunk 2 above)
Comment (raimonesteve) [reply message.xml]: 1ba7bdee1f26e4dd57197db15920ff3fb79033ee

**Pull request merged**